www/src/docs/api/LabelListAPI.tsx

```diff
@@ -0,0 +1,165 @@
+import { ApiDoc } from './types';
+
+export const LabelListAPI: ApiDoc = {
+  name: 'LabelList',
+  props: [
+    {
+      name: 'angle',
+      type: 'number',
+      isOptional: true,
+      desc: {
+        'en-US': (
+          <section>
+            <p>
+              Text rotation angle in degrees. The text will be rotated around the (x, y) coordinates as the pivot point.
+              Positive values rotate clockwise, negative values rotate counterclockwise.
+            </p>
+          </section>
+        ),
+      },
+      defaultVal: 0,
+    },
+    {
+      name: 'clockWise',
+      type: 'boolean',
+      isOptional: true,
+      desc: {
+        'en-US': (
+          <section>
+            <p>The parameter to calculate the view box of label in radial charts.</p>
+          </section>
+        ),
+      },
+    },
+    {
+      name: 'content',
+      type: 'ReactNode | Function',
+      isOptional: true,
+      desc: {
+        'en-US': (
+          <section>
+            <p>
+              If set a React element, the option is the customized React element of rendering each label. If set to a
+              function, the function is called once for each item
+            </p>
+          </section>
+        ),
+      },
+      format: ['<LabelList content={CustomizedLabel} />'],
+      examples: [{ name: 'Customized content of LabelList in a BarChart', url: '/examples/BarChartWithMinHeight/' }],
+    },
+    {
+      name: 'dataKey',
+      type: 'string | number | Function',
+      isOptional: true,
+      desc: {
+        'en-US': (
+          <section>
+            <p>Decides how to extract the value of each label from the data:</p>
+            <ul>
+              <li>
+                <code>string</code>: the name of the field in the data object;
+              </li>
+              <li>
+                <code>number</code>: the index of the field in the data;
+              </li>
+              <li>
+                <code>function</code>: a function that receives the data object and returns the value of each label.
```

Comment:
eventually would be nice to add support for what the types look like for functions I think

Reply:
I was looking for balance here, between information and too much detail. Whoever is interested in the exact type can open the IDE and hover over the prop.